recipes/CommonVoice/common_voice_prepare.py

```diff
@@ -309,8 +314,14 @@ def create_csv(
         raise FileNotFoundError(msg)
 
     # We load and skip the header
-    loaded_csv = open(orig_tsv_file, "r").readlines()[1:]
-    nb_samples = len(loaded_csv)
+    csv_lines = open(orig_tsv_file, "r").readlines()
+    header_line = csv_lines[0]
+    csv_data_lines = csv_lines[1:]
+    nb_samples = len(csv_data_lines)
+
+    header_map = {
+        column_name: index for index, column_name in enumerate(header_line)
+    }
```

Comment:
Doesn't `csv.DictReader` handle all of this more elegantly?

Reply:
As discussed yesterday, will review/test as is and not block on this